src/transforms/rename.jl

```diff
@@ -0,0 +1,43 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+""" 
+    Rename(Dict(:col₁ => :newcol₁, :col₂ => :newcol₂, ..., :col₁ => :newcolₙ))
+
+Tha transform that renames `col₁` to `newcol₁`, `col₂` to `newcol₂`, ...
+""" 
+struct Rename3 <: Stateless
+    names::Dict{Symbol,Symbol}
+end
+
+function apply(transform::Rename3, table)
+    new_table = (;)
```

Comment:
Before we start applying the transform, we need to make sure that the input table has the columns specified in `Rename`.